gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/orchestration_stack.rb

```diff
@@ -21,6 +23,14 @@ def raw_update_stack(template, options)
     raise MiqException::MiqOrchestrationUpdateError, err.to_s, err.backtrace
   end
 
+  def self.fix_options(options)
```

Comment:
@gmcculloug It actually was the (similar) name that I initially came up with. I made it generic because we may have other option to fix. For example, according to the doc 
```
  disable_rollback: false,
  on_failure: "DO_NOTHING", # accepts DO_NOTHING, ROLLBACK, DELETE
```
`disable_rollback` is backward compatible to v1, `on_failure` is for v2. It seems we don't need to immediately convert it.